testing/multi-env-test-engine/src/main/java/org/projectnessie/junit/engine/MultiEnvTestExtension.java

```diff
@@ -37,4 +37,13 @@ public interface MultiEnvTestExtension extends Extension {
    * invocations of this method to ensure a stable test case creation order.
    */
   List<String> allEnvironmentIds(ConfigurationParameters configuration);
+
+  /**
+   * Allows {@link MultiEnvTestExtension}s to define their relative ordering within a JUnit
+   * UniqueID. Higher numbers will appear earlier in the JUnit UniqueId. For extensions with the
+   * same order value, the segmentType will be sorted alphabetically.
+   */
+  default int getOrder() {
```

Comment:
How about `priority()` or `weight()`? 

1) the `get` prefix does not align with the `allEnvironmentIds()` method.
2) "order" sounds a bit too strong. This value affects the placement of the extension's segment in the test `UniqueId` values... it does not affect execution order of tests or extensions at all (cf. the JUnit5 `@Order` annotation).